cnd/src/swap_protocols/herc20/connector.rs

```diff
@@ -0,0 +1,121 @@
+use crate::{
+    asset::{ethereum::FromWei, Erc20, Erc20Quantity},
+    btsieve::ethereum::{
+        watch_for_contract_creation, watch_for_event, Cache, Event, Topic, Web3Connector,
+    },
+    ethereum::{Hash, U256},
+    swap_protocols::{
+        herc20::{data, Deployed, Funded, Params, Redeemed, Refunded},
+        rfc003::Secret,
+    },
+};
+use std::cmp::Ordering;
+use tracing_futures::Instrument;
+
+lazy_static::lazy_static! {
+    static ref REDEEM_LOG_MSG: Hash = blockchain_contracts::ethereum::rfc003::REDEEMED_LOG_MSG.parse().expect("to be valid hex");
+    static ref REFUND_LOG_MSG: Hash = blockchain_contracts::ethereum::rfc003::REFUNDED_LOG_MSG.parse().expect("to be valid hex");
+    /// keccak('Transfer(address,address,uint256)')
+    static ref TRANSFER_LOG_MSG: Hash = "ddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef".parse().expect("to be valid hex");
```

Comment:
Should we update `blockchain_contracts::ethereum::rfc003:` and export it from there? Feels weird to have it in here.

Reply:
Currently compiling so I can't change branch but not sure. Do we have this information in blockchain-contracts? Or are you saying we should add it to the blockchain-contracts crate?

Reply:
Sorry for not being clear: I meant that *if* we don't have it in blockchain-contracts then we should add it there. 
I didn't verify if this is not the case.

Reply:
Yes I see. It's not really something introduced by our protocols contrary to the other values but it is something that belongs to the blockchain contract domain so yes, I think it's a good idea.